test_harness/reps.go

```diff
@@ -15,12 +15,11 @@ type Scenario struct {
 
 type TestCase struct {
 	ExpectedValue interface{}   `json:"expectedValue"`
-	ExpectError   bool          `json:"expectError"`
```

Comment:
Removed because this code will end up only being used in the integration harness where we can't typically check for errors like the ones we were expecting.